documentation/modules/rn-2.2.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
```

Comment:
This file should not appear in the PR because it is already in main. Did you start with a rebased main?